CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Nothing should go in this section, please add to the latest unreleased version
   (and update the corresponding date), or add a new version.
 
+## [1.18.2] - 2022-08-05
```

Comment:
this should be under 1.8.0 since that is the pre released version, i created a pr to fix this, fee lfree to squash it in your stuff.

Reply:
Updated changes to go under 1.18.1